### DISCUSSION TITLE
world hopper, skip speedrunning/special worlds option

### DISCUSSION THREAD

**Discussion by Jin-Jiyunsun** [runelite/runelite#15673] [Ideas]
Id like to suggest a checkbox that once ticked would skip over any special non standard worlds such as speedrunning worlds or league worlds (when leagues are running) as using the world hopper to cycle worlds in sequence at the moment causes you to get stuck on a speedrunning world eventually and defeats the purpose of using the quick hop shortcuts if you have to dig into the menus to manually get past said world.

**Comment by dzcampgirl** [comment]
Love this idea so much.
